src/nvim/syntax.c

```diff
@@ -1620,42 +1615,37 @@ get_syntax_attr(
 /*
  * Get syntax attributes for current_lnum, current_col.
  */
-static int
-syn_current_attr(
-    int syncing,                           // When 1: called for syncing
-    int displaying,                        // result will be displayed
-    bool *can_spell,                       // return: do spell checking
-    int keep_state                         // keep syntax stack afterwards
+static int syn_current_attr(
+    const bool syncing,                   // When true: called for syncing
+    const bool displaying,                // result will be displayed
+    bool *const can_spell,                // return: do spell checking
+    const bool keep_state                 // keep syntax stack afterwards
```

Comment:
In the future, let's not mark non-pointer parameters with `const`. It's mostly noise.

I know there's debate about it, but we need to have a clear guideline about this, and it's better if we just don't do it.